shared/params/const.go

```diff
@@ -0,0 +1,11 @@
+package params
+
+type KeyBytes = [48]byte
```

Comment:
This should be `PublicKey` not just `Key`.

Reply:
48 bytes is also the private key no?